### DISCUSSION TITLE
如何在任务栏显示动态图标？

### DISCUSSION THREAD

**Discussion by tensor-tech** [linuxdeepin/developer-center#5119] [Development Questions | 开发答疑]
类似windows 任务栏 的 天气 一样，如何在deepin 任务栏显示 一个动态的图片呢？

**Comment by BLumia** [comment]
写一个 dock 插件即可，参见 dde-dock 自带的插件、 https://github.com/sonichy/WEATHER_DDE_DOCK 或者其它插件项目。
